### ISSUE TITLE
Pointgroup analyzer returns wrong point group

### ISSUE THREAD

**Issue by JPDarby** [materialsproject/pymatgen#664]
<When reporting bugs/issues, please supply the following information. If
this is a feature request, please simply state the requested feature.>

## System

* Pymatgen version: <4.7.2>
* Python version: <3.6.1>
* OS version: <OpenSUSE 13.1> 

## Summary

* <If you pass 8 carbon atoms at the corners of a cube to the point group analyzer it does not detect the 4 fold rotation axes and returns the incorrect point group. >

## Example code

```python
<Example code generating error>
```

## Error message

```
<Error message>
```

## Suggested solution (if any)

* <If you have a suggestion on how to solve the issue, you may write it
  here.>

## Files (if any)

<If input files are needed to reproduce the error, please provide 
either links (e.g., Dropbox or GDrive) for large files or simply paste
the file below for small files.>

```
<contents of file 1>
```

**Comment by shyuep** [comment]
Can you provide an example code?

**Comment by JPDarby** [comment]
[point_group_example.txt](https://github.com/materialsproject/pymatgen/files/992427/point_group_example.txt)

**Comment by shyuep** [comment]
Thanks for the report. I have fixed the bug (it is actually not due to bad detection of rotation axes, but rather wrong classification of a cube as a symmetric top instead of a spherical top due to the fact that the center of mass is not at the origin during calculation of the inertia tensor). I have released v4.7.6 which contains this fix as well as other minor fixes and improvements.
